openshift/4.0/templates/Logging/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.5"
+
+services:
+  logging:
+    restart: on-failure:1
+    container_name: pims-logging
+    build:
+      context: .
+    image: pims-sidecar:latest
```

Comment:
The image comes from where?

Reply:
The image is a 5Mb alpine image from the docker file tagged to pims-sidecar after build

Reply:
If I understand correctly.  We would need to manually build the docker file and tag it `pims-sidecar`.  You may want to follow the practice of pointing the `docker-compose.yml` to the `Dockerfile` instead.  Use the `build: context:`.